source/api/image/3.0/change-log.md

```diff
@@ -0,0 +1,158 @@
+
+
+## 3. Editorial Changes
+
+The following changes do not change the Image API behavior from version 2.1.1.
+
+#### 3.1. Clarify distinctions between underlying image content, full image, extracted region, and image returned
+
+The [Terminology][image30-terminology] section now explicitly defines **underlying image content** and **full image**. These terms, along with extracted region and image returned are used to describe the image manipulations more consistently than in previous versions. See issue [#1425](https://github.com/IIIF/api/issues/1425).
+
+#### 3.2. Clarify description of the _`!w,h`_ form for the size parameter
+
+Description of the _`!w,h`_ form for the size parameter has been clarified to point out that the returned image must be as large as possible to fit within the _`w,h`_ box subject to constraints of extracted region size and server-imposed limits. See issue [#1372](https://github.com/IIIF/api/issues/1372).
+
+#### 3.3. Clarify that the `color` format value might still yield a non-color image
+
+Description of the `color` value of the format parameter has been clarified to make it clear that it is a request fro the image with all of its color information. If the underlying image content has no color information then the resulting image will not have any either, even with the `color` format value. See issue [#1375](https://github.com/IIIF/api/issues/1375) and [#1435](https://github.com/IIIF/api/issues/1435).
+
+#### 3.4. Changed examples to use `https` URIs
+
+All examples now use `https` URIs in order reflect best practices for interoperable implementations. See issue [#1421](https://github.com/IIIF/api/issues/1421).
+
+#### 3.5. Clarify and move discussion of floating point representation
+
+Discussion of floating point representation has been moved from the image requests introduction to the [Canonical URI Syntax][image30-canonical-uri-syntax] section, and has been expanded for clarity. The advice on using at most 10 decimal digits has been removed. See issue [#1468](https://github.com/IIIF/api/issues/1468).
+
+#### 3.6. Clarify support for CORS
+
+Discussion of CORS has be moved to a new [CORS][image30-cors-response] section and the wording clarified to make it clear that not only the `Access-Control-Allow-Origin` header but also the preflight request pattern (ie. including the `OPTIONS` header) should be supported. See issue [#1274](https://github.com/IIIF/api/issues/1274).
+
+#### 3.7. Implementation with static web resources
+
+Add explicit mention of support for implementation with pre-generated static files.
+
+#### 3.8. Font consistency
+
+The use of `code` font and capitalization was made consistent for class names, property names and values when used in prose.
+
+## 4. Related Document Changes
```

Comment:
We should re-normalize with Prezi. Either they both have the section or neither.

Reply:
:+1: removing for now to match... put text about registry in 2.1